### PR TITLE
Put solutions in folder, fix master branch error

### PR DESCRIPTION
**Pull request by m4v15**
Solves #18 #5

### CODE REVIEW

**Comment by m4v15** [comment]
@jsms90 @Akin909 @eliascodes not sure who to assign to this as no author or maintainer, assigning you three along with some nazarenes as akin made it (I think), and as two other CF (figure elias might want to use this for Gaza next week.

File: server/server.js [hunk 1]
```diff
@@ -2,11 +2,14 @@
 const express = require('express');
 const path = require('path');
 const routes = require('./routes');
+const bodyParser = require('body-parser')
 
 const app = express();
 
 //TODO We need middleware to convert request and response payloads to an from JSON
+app.use(bodyParser.json())
```
Comment (eliascodes) [inline server/server.js]: Why have these todo's been filled in? Is this not part of the exercise?

> Your first task should be filling in the missing middleware that your server needs. See the comments in server.js for hints.

Comment (m4v15) [inline server/server.js]: Oh sorry, forgot to move them into the solution folder

Comment (m4v15) [inline server/server.js]: @eliascodes this should be fixed now. I could restructure the solution folder to have just a copy of the whole server folder with the files that are different changed, but I feel like it's probably this sort of thing that will lead to the same problem that this PR is trying to fix (things being different in files the students aren't expected to change in a solution)

**Comment by mantagen** [comment]
@Akin909 you good for this to merge? Delivering it tomorrow morning so would be ace for it to be in then... Cheers matey

**Pull request merged**